src/geometry/closest_points.rs

```diff
@@ -42,12 +42,12 @@ fn closest_points_aux(
 
     if n <= 3 {
```

Comment:
Shouldn't this be `n >= 3`?

Reply:
Nope, if `n >= 3` the simple bruteforce is too expensive.
It's a recursive algorithm where all array of points is recursive checked and the tail of the recursion is the case when points less or equal than 3.